pkg/aggregator/aggregator_test.go

```diff
@@ -1170,13 +1194,18 @@ definitions:
 `), &expected)
 
 	ast := assert.New(t)
-	if !ast.NoError(MergeSpecs(spec1, spec2)) {
+	orig_spec2, _ := cloneSpec(spec2)
```

Comment:
Is there anything in here already that verifies cloneSpec works correctly?  A bug in cloneSpec would silently fail some of these tests.

Reply:
It's a basic `json.Marshal+Unmarshal`. Not so much that can go wrong. But I can add a test of course.

Reply:
added a test